feat: Automatic Scaling for Isometric tiles and Negative indexed tile access (#1463)
flame-engine/flame: doc/flame/components.md

![An example of a isometric map with selector](../images/isometric.png)

> The component Also supports negative indexing starting from -2 *(as -1 is reserved for empty tile)* which returns sprite from bottom Left to bottom right using negative Indexing and wraps to upper rows on overflow of tileset's image width. 

Reviewer: ```suggestion
The component supports negative indexing starting from -2 *(as -1 is reserved for empty tile)* 
which returns the sprite from the bottom left to the bottom right using negative indexing and
wraps to the upper rows on overflow of the tileset's image width. 
```

> Use `propSize` Property to give a custom Size to negative Indexed elements *(This can be used to add props and art as Tiles which rise higher than most tiles)*.

Reviewer: ```suggestion
Use the `propSize` property to give a custom size to negative indexed elements 
*(This can be used to add props and art such as tiles which rise higher than most tiles)*.
```

> An Example of rendering Props as tiles is something like those mirrors some of which are higher than normal tiles

Reviewer: ```suggestion
The following is an example of rendering props as tiles. In this example it is specifically the
mirrors, some of which are higher than the normal tiles.
```
